Security/SessionDownloadStrategy.php

```diff
@@ -14,12 +14,15 @@
 use Sonata\MediaBundle\Model\MediaInterface;
 use Symfony\Component\DependencyInjection\ContainerInterface;
 use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Session\Session;
 use Symfony\Component\Translation\TranslatorInterface;
 
 class SessionDownloadStrategy implements DownloadStrategyInterface
 {
     /**
      * @var ContainerInterface
+     *
+     * @deprecated Since version 3.x, will be removed in 4.0.
```

Comment:
Please add the `NEXT_MAJOR` here too (basically, everywhere we must not forget to do something).